libs/@guardian/source-react-components/src/button/styles.ts

```diff
@@ -303,8 +303,8 @@ export const buttonStyles =
 		button,
 		sizes[size],
 		priorities[priority](theme.button),
-		iconSvg || isLoading ? iconSizes[size] : '',
-		(iconSvg || isLoading) && !hideLabel ? iconSides[iconSide] : '',
+		iconSvg ?? isLoading ? iconSizes[size] : '',
+		(iconSvg ?? isLoading) && !hideLabel ? iconSides[iconSide] : '',
```

Comment:
if iconSvg is `false`, `0` or `""`  ```iconSvg ?? isLoading``` will now evaluate as false

Reply:
I _think_ thats what we want right? This was the linters auto fix iirc

Reply:
Yea i think it doesn't matter because iconSvg has type `iconSvg: React.ReactElement<any, string | React.JSXElementConstructor<any>> | undefined` so it can't be any of those values

Reply:
my hunch is that we were just using `||` (or now `??`) as a catch all for any falsy value